requirements-dev.txt

```diff
@@ -8,7 +8,7 @@ nose_parameterized==0.3.3
 profanityfilter
 lxml==4.5.0
 splinter>=0.13.0
--e git+https://github.com/qld-gov-au/ckanext-data-qld@develop#egg=ckanext-data_qld
+-e git+https://github.com/salsadigitalauorg/ckanext-data-qld@feature/Enhancements_to_the_data_request_functionality#egg=ckanext-data_qld
```

Comment:
sometimes i think it would be awesome to allow these to be dynamic (group name) as well as switching branch/tag with pipeline parallel builds so that we can test against develop/master/latest tagged release etc.